src/MacroExpander.js

```diff
@@ -108,6 +108,32 @@ export default class MacroExpander implements MacroContextInterface {
         this.stack.push(...tokens);
     }
 
+    /**
+     * Find an macro argument without expanding tokens and append the array of
+     * tokens to the token stack. Uses Token as a container for the result.
+     */
```

Comment:
If the argument is a group will the contents of the return token have to be reparsed?

Reply:
@kevinbarabash They're not parsed here, but only fetched (buffered).